src/components/beneficiary/CreateForm.tsx

```diff
@@ -0,0 +1,180 @@
+      companyId: values.type == BeneficiaryType.company ? values.companyId : undefined,
+      countryCode:
+        companies?.find((c) => c.id === values.companyId)?.countryCode || values.countryCode,
+      cityId: companies?.find((c) => c.id === values.companyId)?.cityId || values.cityId,
+      organizerRelation: values.organizerRelation,
+      description: values.description,
+      campaigns: values.campaigns,
+    }
+    mutation.mutateAsync(data)
+  }
+
+  return (
+    <GenericForm
+      onSubmit={onSubmit}
+      initialValues={initialValues}
+      validationSchema={validationSchema}>
+      <Box sx={{ height: '62.6vh', marginBottom: '9%' }}>
+        <Typography variant="h5" component="h2" sx={{ textAlign: 'center' }}>
+          {t('beneficiary:forms:add-heading')}
+        </Typography>
+        <Typography variant="body1" sx={{ textAlign: 'center', padding: '8px', mb: 2 }}>
+          {t('beneficiary:forms:info')}
+        </Typography>
+        <Grid container spacing={3}>
+          <Grid item xs={6}>
+            <BeneficiaryTypeSelect name={'type'} setBeneficiaryType={setBeneficiaryType} />
+          </Grid>
+          <Grid item xs={6}>
+            <OrganizerRelationSelect
+              name="organizerRelation"
+              label="beneficiary:grid:organizerRelation"
+            />
+          </Grid>
+          {beneficiaryType === BeneficiaryType.individual ? (
+            <>
+              <Grid item xs={12}>
+                <Typography paddingLeft={'inherit'} marginBottom={2}>
+                  {t('beneficiary:forms.labels.person-select')}{' '}
+                  <Link href={routes.admin.person.create}>
+                    {t('beneficiary:forms.labels.create-new')}
+                  </Link>
+                </Typography>
+                <PersonSelect name="personId" label={t('beneficiary:forms.labels.person-label')} />
+              </Grid>
+              <Grid item xs={6}>
+                <CountrySelect name="countryCode" />
+              </Grid>
+              <Grid item xs={6}>
+                <CitySelect name="cityId" />
+              </Grid>
+            </>
+          ) : (
+            <Grid item xs={12}>
+              <Typography paddingLeft={'inherit'} marginBottom={2}>
+                {t('beneficiary:forms.labels.company-select')}{' '}
+                <Link href={routes.admin.company.create}>
+                  {t('beneficiary:forms.labels.create-new')}
+                </Link>
+              </Typography>
+              <CompanySelect name="companyId" label={t('beneficiary:forms.labels.company-label')} />
+            </Grid>
+          )}
+          <Grid item xs={12}>
+            <FormTextField
+              type="text"
+              name="description"
+              autoComplete="target-amount"
+              label={t('beneficiary:grid:description')}
+              multiline
+              rows={1.5}
+            />
```

Comment:
the autoComplete here seems not correct, please remove as it doesn't make sense for description field
also lets have rows=2